### DISCUSSION TITLE
Shortcut to delete annotations

### DISCUSSION THREAD

**Discussion by phanen** [sumatrapdfreader/sumatrapdf#3069] [Q&A] [answered]
Is there anyway to delete annotations using shortcut?

I often select the texts and type `A` to create annotations.

Is there any shortcut to delete annotations? For example:
- Select the texts and type `somekey` to delete annotations.
- Or any shortcut to delete the recently created annotations.

**Comment by GitHubRulesOK** [comment, marked as answer]
in current pre-rel version the shortcut is `Delete` key however that at first may only remove display of colour (the tooltip for that area may still be active in memory)
Generally, not a good idea to remove annotations as they still reside invisibly in the file, thus often take up resources, The way adobe designed PDF to not need full editing is notes/highlights are like stickers on the page, but there is no delete ability, so delete simply changes their appearance, by means of dis-appearance.
